.rubocop_todo.yml

```diff
@@ -971,6 +971,7 @@ Style/PerlBackrefs:
 Style/PredicateName:
   Exclude:
     - 'spec/**/*'
+    - 'src/api/app/controllers/webui/kiwi/images_controller.rb'
```

Comment:
Many people in the rubocop issue discussion are making a point that I agree with: why do we want a method in a controller that starts with `is_`? Controller actions should follow the REST pattern: show,edit,new,update,create,destroy.